_std/pathfinding.dm

```diff
@@ -450,6 +450,10 @@
 			if (istype(A, /obj/overlay) || istype(A, /obj/effects)) continue
 			if ((passer || id) && A.density)
 				if (O.object_flags & BOTS_DIRBLOCK) //NEW - are we a door-like-openable-thing?
+					if(istype(O, /obj/machinery/door))
+						var/obj/machinery/door/door = O
+						if(!door.operating)
+							return FALSE
```

Comment:
```suggestion
					if(istype(O, /obj/machinery/door))
						var/obj/machinery/door/door = O
						if (door.isblocked())
							return FALSE
```
operating doesn't really do this.
I believe isblocked() should be the correct proc to call to check. I tested it locally and it seems to work well this way!